### ISSUE TITLE
IE10,11, Super expression must either be null or a function, not object

### ISSUE THREAD

**Issue by SerkanSipahi** [google/traceur-compiler#1946]
Is there any bug in IE10,11 in Traceur? In all other Browsers(Chrome, Firefox, Safari8) our Application, where we developing for 6 month, works well!

We get following error:

```error
Super expression must either be null or a function, not object.
   at getProtoParent ( Unknown script code:951:5)
   at createClass  ( Unknown script code:941:7)
   at Anonymous function ( eval code:32:9)
   at execute ( eval code:28:7 )
```

**Comment by arv** [comment]
Do you happen to know what value your extends expression is?

**Comment by SerkanSipahi** [comment]
Our class extends from HTMLElement

e.g. 

```js
   class foo extends from HTMLElement {

   }
```

**Comment by SerkanSipahi** [comment]
I believe its related to https://github.com/google/traceur-compiler/issues/1709 :(

**Comment by arv** [comment]
Yup. `typeof HTMLElement` is incorrectly reporting `'object'`

**Comment by SerkanSipahi** [comment]
@arv Do you have any idea how we can resolve this problem?

**Comment by arv** [comment]
Let's move the discussion to #1709
